src/utrfx/variant_util.py

```diff
@@ -129,3 +144,26 @@ def retrieve_variants_of_region(
             for alt in rec.alts:
                 variant_list.append(VariantCoordinates.from_vcf_literal(contig=contig, pos=pos, ref=ref, alt=alt))
         return variant_list
+    
+    def get_allele_frequency(
+        self,
+        variant: VariantCoordinates,
+    ) -> typing.Optional[float]:
+        """
+        Get the allele frequency for the specified variant.
+
+        :param variant: single variant as `VariantCoordinates`class instance.
+        """
+        assert self._vcf_file is not None, "VCFfile must be used as a context manager"
+        contig = f"chr{variant.chrom}"
+        for rec in self._vcf_file.fetch(contig, variant.start, variant.end):
+            af_tuple = rec.info.get('AF', None)
+            if len(af_tuple) == 1:
+                return af_tuple[0] 
```

Comment:
I think this is incorrect.

Suppose query = `chr1:123A>C` while GnomAD includes AF `chr1:123A>T`. This code will report AF for the incorrect allele!